docs/docsite/rst/dev_guide/developing_inventory.rst

```diff
@@ -1,21 +1,220 @@
 .. _developing_inventory:
 
```

Comment:
Overall this looks good. I made a few notes about typos and places where I think the documentation could be made even stronger. Thanks for regularizing the header notation.